game-core/src/main/java/games/strategy/triplea/ui/EditPanel.java

```diff
@@ -953,19 +953,19 @@ private static void sortUnitsToRemove(final List<Unit> units) {
 
   private static Comparator<Unit> getRemovableUnitsOrder() {
     return Comparator.comparing(
-        TripleAUnit::get,
+        Function.identity(),
```

Comment:
Well in this case there's no point to use the surrounding `Comparator.comparing`, just return the lambda directly I'd say

Reply:
Agree, updated in: a82b9830a